orc8r/cloud/Makefile

```diff
@@ -151,6 +152,26 @@ swagger_directories:
 	rm -rf $(SWAGGER_V1_SPECS_DIR)
 	mkdir -p $(SWAGGER_V1_COMMON_DIR) $(SWAGGER_V1_PARTIAL_SPECS_DIR) $(SWAGGER_V1_STANDALONE_DIR)
 
+nms_node:
+	# install pre-reqs
+	apt update
+	apt install -y nodejs npm
+	# symlink nodejs executable
+	mkdir -p /usr/local/bin
+
+nms_gen: nms_node
+	NODE="$(shell which nodejs)"; ln -s $$NODE /usr/local/bin/node
```

Comment:
Move all the "install" steps to the nms_node -- that way we can optionally only run that once then re-run the nms_gen multiple times faster